docs/source/performance.md

```diff
@@ -213,74 +209,74 @@ Suffice to say large payload sizes are an anti-pattern in any blockchain solutio
 
 ### Chaincode language
 
-Go chaincode performs best, followed by Node chaincode.  Java chaincode performance is the least performant and would not be recommended for high throughput applications.
+Go chaincode has shown the best performance in Hyperledger Fabric environments, followed by Node chaincode. Java chaincode performance is the least performant of the three, and is not recommended for high-throughput applications.
 
 ### Node chaincode
 
-Node is an asynchronous runtime implementation that utilises only a single thread to execute code. It does however run background threads for activities such as garbage collection, however when allocating resources to a Node chaincode, for example in Kubernetes where you are limited to available resources, it doesn't make sense to allocate multiple vCPUs for Node chaincode. The number of vCPUs in a system usually refers to the number of concurrent threads that can be executed. It is worth monitoring performance of Node chaincode to see how much vCPU it uses but it probably doesn't make sense to allocate anything more than a max of 2 vCPUs for Node chaincode. In fact you could not assign any resource restrictions to Node chaincode as it is self limiting.
+Node is an asynchronous runtime implementation that utilizes only a single thread to execute code. Node does run background threads for activities such as garbage collection, but it is against best practices to allocate more than two vCPUs (generally equivalent to the number of concurrent threads which can be executed) to a Node chaincode (for example on Kubernetes which is limited to available resources). It is worth monitoring performance of Node chaincode to see how much vCPU it uses, though you cannot assign resource restrictions to Node chaincode as it is self-limiting.
```

Comment:
```but it is against best practices to allocate more than two vCPUs``` I think this is too harsh a statement. It's not against best practices really and it may be that you can get slightly more performance if you do allocate more than 2 vCPUs because node can run other threads concurrently but it might not be worth the cost to do so for the little gain there is. So I would still prefer to emphasise that node is generally self-limiting and that there may not be much value allocating more than 2 vCPUs

Reply:
So the point is you don't need to assign resource restrictions bec. Node is self limiting? Is that the guidance.

Reply:
"Node chaincode is self-limiting and not unbounded, though you could also assign resource restrictions to Node chaincode processes."